GrootCI.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
```

Comment:
I don't see such a file in this project, maybe you need to add one?

Note that a plain `requirements.txt` file usually leads to non-reproducible test and builds, consider using other tools for dependency management such as [pipenv](https://pipenv.pypa.io/en/latest/), [poetry](https://python-poetry.org/) or [pip-tools](https://pypi.org/project/pip-tools/)

Reply:
Thank's for the input, the requirements.txt file is found in another PR (there has been some sort of miscommunication on how atomic the changes need to be in each PR so it can be found in the PR#40

Reply:
well, unfortunately GitHub does not implement the concept on PR dependencies at this point, so your team needs to agree on a way to signal such dependencies so people know not to merge this PR before the other one.

Also consider that if the CI was actually running, it would fail for this PR.

Reply:
@flippy1989 Have you considered @ifireball 's recomendations?

Reply:
The only reason I have the requirements in another PR is that you instructed me to do so @lioramilbaum